tests/GridFS/StreamWrapperFunctionalTest.php

```diff
@@ -47,7 +47,7 @@ public function testReadableStreamRead()
 
         $this->assertSame('abc', fread($stream, 3));
         $this->assertSame('defghij', fread($stream, 10));
-        $this->assertSame('', fread($stream, 3));
+        $this->assertNull(fread($stream, 3));
```

Comment:
Does this actually pass for you?

Running this on my end, I see `Failed asserting that '' is null.` [`fread()`](http://php.net/fread) is documented as returning either a string or `false` on failure, but never `null`.